### PR TITLE
Datomic and deps (memcached and postgres)

### PR DESCRIPTION
**Pull request by bombaywalla**
This PR should install `memcahced`, `postgres`, and `datomic-pro`.
Tested on Vagrant. Not tested on Exoscale.
I have checked in the roles downloaded from Ansible Galaxy. Hope that is the right thing to do.
I have copied over (and made a small bugfix to) the role `plexus.datomic-pro` from `plexus/ansible_playbooks`

### CODE REVIEW

**Review by lispyclouds — APPROVED** [review]

**Comment by lispyclouds** [comment]
@plexus would you have a look too?

File: clojurians_log/playbooks/roles/geerlingguy.memcached/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - distro: ubuntu1404
+  - distro: ubuntu1204
+
+script:
+  # Configure test script so we can run extra tests after playbook is run.
+  - export container_id=$(date +%s)
+  - export cleanup=false
+
+  # Download test shim.
+  - wget -O ${PWD}/tests/test.sh https://gist.githubusercontent.com/geerlingguy/73ef1e5ee45d8694570f334be385e181/raw/
```
Comment (victorb) [inline clojurians_log/playbooks/roles/geerlingguy.memcached/.travis.yml:16]: Might want to check the sha1sum of this downloaded file before running it.

Comment (lispyclouds) [reply clojurians_log/playbooks/roles/geerlingguy.memcached/.travis.yml:16]: This is from a vendored dependency. Don't think @bombaywalla has written it, was there as it is before.

**Comment by lispyclouds** [comment]
Merging this.

**Pull request merged**